packages/aws-signing-helper/aws-signing-helper.spec

```diff
@@ -0,0 +1,49 @@
+%global goproject github.com/aws
+%global gorepo rolesanywhere-credential-helper
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.0.2
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+%global gitrev b193b4e041c18896203b77724fa0577cf73949fa
+%global shortrev %(c=%{gitrev}; echo ${c:0:7})
```

Comment:
nit: neither of these variables are used, but we can clean them up next time there's an update